apps/wallet-connect/src/hooks/useWalletConnect.tsx

```diff
@@ -117,6 +117,19 @@ const useWalletConnect = () => {
               result = '0x'
               break
             }
+
+            case 'eth_signTypedData_v4': {
```

Comment:
let's also add support for `eth_signTypedData`, version postfix is metamask specific and not a part of the official eip

Reply:
Updated ;)